source/plugins/go/src/oms.go

```diff
@@ -2361,6 +2381,8 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 	} else if IsWindows && IsAADMSIAuthMode { // AMA windows specific
 		Log("Creating AMA client for KubeMonAgentEvents")
 		EnsureGenevaOr3PNamedPipeExists(&KubeMonAgentEventsNamedPipe, KubeMonAgentEventDataType, &KubeMonEventsWindowsAMAClientCreateErrors, false, &MdsdKubeMonAgentEventsTagRefreshTracker)
+		Log("Creating AMA client for InsightsMetrics")
+		EnsureGenevaOr3PNamedPipeExists(&InsightsMetricsNamedPipe, InsightsMetricsDataType, &ContainerLogsWindowsAMAClientCreateErrors, false, &MdsdContainerLogTagRefreshTracker)
```

Comment:
we should track telemetry separately for  InsightsMetrics instead of reusing ContainerLogsWindowsAMAClientCreateErrors